packages/Datepicker/src/datepicker.scss

```diff
@@ -0,0 +1,149 @@
+@use '~@igloo-ui/tokens/dist/base10/variables' as tokens;
+@use '~@igloo-ui/tokens/dist/fonts';
+
+$cell-size: tokens.$space-6;
+$cell-number: 7;
+$cell-border: calc(2 * 1px);
+$cell-empty-border: calc(6 * 0.5px);
```

Comment:
This intrigues me, what is the benefit of writing these values ​​this way? Because at the end, `calc(2 * 1px)` and `calc(6 * 0.5px)` are constantly giving `2px` and `3px`, it's not like using relative scaling like `%`

Reply:
The calculation is done to obtain a square cell for the day display. I have separated the elements to make the calculation easier to read. I will add comments to make it clear what each variable does.

Reply:
Fixed